app/views/games/edit.html.erb

```diff
@@ -1,8 +1,34 @@
-<h1>Editing game</h1>
+<h1 style="text-align: center">Game <%= @game.id.split("-").first %></h1>
 
-<%= render "form", game: @game %>
-<br/>
-<div class="btn-group">
-  <%= link_to "Show", @game, class: "btn btn-info btn-sm" %>&nbsp;
-  <%= link_to "Back to games", games_path, class: "btn btn-info btn-sm" %>&nbsp;
-</div>
+<table class="table table table-striped table-hover table-sm">
```

Comment:
I don't know about having a table for update/show 🤔 But we can keep it as is for now, and change in future if we need to

Reply:
I was thinking that every game will apear as a table with rounds and scores, this is why a chose this type of styling. Of course we will change it if we see fit when it will have more data.